### ISSUE TITLE
txn went through but the eth is not showing up in etd smart contract window

### ISSUE THREAD

**Issue by Maldavies** [etherdelta/etherdelta.github.io#352]
0x06ea8a820ece42754c6c08bf2676f159b026047b0571b01e7f74bbe4e3fc57a7

Any help appreciated.

Mal

**Comment by vaibhavchellani** [comment]
yeah i checked that your transaction has indeed occured on the blockchain , no need to worry try refreshing the page , probably an issue with your internet connection
